conda_smithy/templates/README.md.tmpl

```diff
@@ -320,8 +320,8 @@ In order to produce a uniquely identifiable distribution:
 
 Feedstock Maintainers
 =====================
-
-{% for maintainer in maintainers -%}
-* [@{{maintainer}}](https://github.com/{{maintainer}}/)
-{% endfor %}
+{# this comment ensures a leading newline #}
+{%- for (maintainer, url) in maintainers|zip(maintainer_urls) %}
```

Comment:
This line does not work because "jinja2.exceptions.TemplateAssertionError: No filter named 'zip'." How does one zip two lists with jinja2?

Reply:
There is this: https://stackoverflow.com/questions/5208252/ziplist1-list2-in-jinja2 but I think it may be easier to zip the two lists before passing the value to the template. You could use a dict that maps maintainer to url and then use the .items() method.